data/css/modules/banner/_app.scss

```diff
@@ -1,21 +1,47 @@
 .BannerApp {
-    font-family: $logo-font;
-    font-size: 1.19em; /* 19px / 12px/em * 72/96 */
-
     &__subtitle {
+        font-size: 1.19em; /* 19px / 12px/em * 72/96 */
         font-family: $support-font;
         color: $primary-light-color;
         letter-spacing: 0.74px;
         margin-top: 70px;
     }
 
-    .ThemeableImage {
+    &__image {
         -EknThemeableImage-sizing: "size-down";
         min-width: 200px;
         min-height: 200px;
     }
 
-    .composite & {
+    .DynamicLogo {
+        -EknDynamicLogo-layout: "vertical";
+        -EknDynamicLogo-mode: "full";
+
+        &__image {
+            color: white;
+            min-height: 100px;
+            min-width: 100px;
+            -EknThemeableImage-sizing: "size-min";
+            -gtk-icon-source: -gtk-recolor(url('resource:///app/assets/logo-symbolic.svg'));
+        }
```

Comment:
I wonder if there is a better way than the obscure symbolic logo feature.
I think if we add a custom text style property to specify the logo uri we could preppend the css to the svg data itself before rendering/loading the image with librsvg.

Long term solution would be to properly integrate svg in GTK